rotkehlchen/chain/evm/decoding/aave/common.py

```diff
@@ -206,18 +212,20 @@ def _decode_withdrawal(
                 )
             ):
                 if (
-                    event.location_label == to and
+                    (event.location_label == to or (event.asset in (A_ETH, A_WETH_SCROLL) and is_weth_user)) and  # noqa: E501
                     event.event_type == HistoryEventType.RECEIVE and
                     event.address != ZERO_ADDRESS
                 ):
                     event.event_type = HistoryEventType.WITHDRAWAL
                     event.event_subtype = HistoryEventSubType.REMOVE_ASSET
-                    event.location_label = user
+                    if is_weth_user is False:
+                        event.location_label = user
```

Comment:
else? Who sets the location label and what is the expected value?

Reply:
The location label is correctly set by the transfer decoder